inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/PdfExtractFile.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2017
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.pdfeditor.pdfanno.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class PdfExtractFile
+{
+
+    private String pdftxt;
+
+    private String stringContent;
+
+    /**
+     * Contains position mapping for a character between PDFExtract string
+     * including and excluding Draw Operations.
+     */
+    private Map<Integer, Integer> stringPositionMap;
+
+    private Map<Integer, PdfExtractLine> extractLines;
+
+    public PdfExtractFile(String pdftxt)
+    {
+        setPdftxt(pdftxt);
+    }
+
+    public void setPdftxt(String pdftxt)
+    {
+        this.stringPositionMap = new HashMap<>();
```

Comment:
Please prefix arguments with `a` and do not use `this.`. Applies in general.

Reply:
Would it be `anId` then or should I go with `a` for all variables?

Reply:
"a" for "argument" not for "a" (determiner) - so not "anId" but "aId"